plugins/slicer/MONAILabel/MONAILabel.py

```diff
@@ -1168,9 +1175,10 @@ def onNextSampleButton(self):
                 "Are you sure to continue?"
             ):
                 return
-            self.onResetScribbles()
             slicer.mrmlScene.Clear(0)
 
+        self.onResetScribbles()
```

Comment:
I am not sure why you are moving this outside the if condition?
Is there a specific logic that requires this?